### ISSUE TITLE
Assumes fenix is added as an overlay

### ISSUE THREAD

**Issue by SuperSandro2000** [NuschtOS/nvim.nix#4]
```
       error: attribute 'fenix' missing
       at /nix/store/50g8n5jhz6faccmigy2xgi07s9hynp5m-source/module/lsp.nix:3:10:
            2| let
            3|   rust = pkgs.fenix.stable.completeToolchain;
             |          ^
            4| in
       Did you mean one of felix, febio, fend, flix or helix?
```

**Comment by SuperSandro2000** [comment]
https://github.com/NuschtOS/nvim.nix/pull/5 has a commit to also fix this [`db88ec0` (#5)](https://github.com/NuschtOS/nvim.nix/pull/5/commits/db88ec055ed322e69cd4672cbf9c43b52b1ed714)

**Comment by SuperSandro2000** [comment]
#6
